.github/workflows/docs.yml

```
@@ -37,3 +37,14 @@ jobs:
 
       - name: Build HTML documentation
         run: make docs-html
+
+  vale:
+    name: runner / vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: errata-ai/vale-action@reviewdog
+        env:
+          # Required, set by GitHub actions automatically:
+          # https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Add final new line to remove the red circle.
```suggestion
          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}

```